openmls/src/lib.rs

```diff
@@ -21,39 +21,40 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![cfg_attr(not(test), forbid(unsafe_code))]
 
+// === Testing ===
+
+/// Single place, re-exporting all structs and functions needed for integration tests
+#[cfg(any(feature = "test-utils", test))]
+pub mod prelude_test;
+
 #[cfg(any(feature = "test-utils", test))]
 pub use rstest_reuse;
 
+#[cfg(any(feature = "test-utils", test))]
 #[macro_use]
-mod utils;
+pub mod test_utils;
+
+// === Modules ===
 
 #[macro_use]
-pub mod error;
+mod utils;
 
-#[cfg(any(feature = "test-utils", test))]
 #[macro_use]
-pub mod test_utils;
+pub mod error;
```

Comment:
Any reason we're not exposing this through `prelude.rs`?

Reply:
We should, I wasn't sure about the macros. Usually they need to be exposed at the root of the crate.

Reply:
They don't need to be, they will be 😉 Macros will always get the high level scope, independent of the module where they are defined.